src/extension.ts

```diff
@@ -1,8 +1,7 @@
 import { ExtensionContext } from "vscode";
 import { GodotTools } from "./godot-tools";
 
-
-let tools: GodotTools = null;
+export let tools: GodotTools = null;
```

Comment:
I think it is not required to export the tools variable because the extension.ts is not designed as a reusable module but the entry point of the plugin.